sky/cli.py

```diff
@@ -681,6 +704,9 @@ def _create_and_ssh_into_node(
         session_manager: Attach session manager: { 'screen', 'tmux' }.
         user_requested_resources: If true, user requested resources explicitly.
         no_confirm: If true, skips confirmation prompt presented to user.
+        idle_minutes_to_autostop: Automatically stop the cluster after this
+                                  many minutes of idleness.
+        retry_until_up: Retry until up when true.
```

Comment:
Improve description, look above.

Reply:
Sounds good! Should I put a condensed description here, or make them just as long as above.

Reply:
Probably a summarized description of the above comment.